spec/system/candidate_interface/application_choices_are_delivered_to_providers_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.feature 'Candidate application choices are delivered to providers' do
+  scenario 'the candidate receives an email' do
+    given_my_application_is_ready_to_send_to_providers
+
+    when_my_application_is_delivered_to_the_provider
+
+    then_i_should_receive_an_email_saying_my_application_is_under_consideration
+  end
+
+  def given_my_application_is_ready_to_send_to_providers
+    @application_choice = create(:application_choice, :ready_to_send_to_provider)
+  end
+
+  def when_my_application_is_delivered_to_the_provider
+    SendApplicationsToProvider.new.call
```

Comment:
Can we make this even higher level by running the sidekiq worker?

Reply:
Good idea. It would be good to have an utterly generic interface to the sweeper task so specs like this could just declare 'the cron has run' and call it.